src/Sarif/AnalyzeContextBase.cs

```diff
@@ -207,6 +216,11 @@ public virtual void Dispose()
                                 "CoreSettings", nameof(OutputFilePath), defaultValue: () => string.Empty,
                                 "The path to write all SARIF log file results to.");
 
+        public static PerLanguageOption<string> PostUriProperty { get; } =
```

Comment:
>PostUriProperty [](http://example.com/codeflow?start=48&length=15)

This kind of change allows this data to be read from/written to our XML driven configuration.